site/src/content/docs/contribute/principles.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Zarf Principles
+---
+
+Zarf is a tool for continuous software delivery to disconnected environments. To ensure that Zarf remains true to its mission and continues to provide value for its users, the following design principles serve as a framework for evaluating new features and capabilities.
+
+## 1. Airgap-First Functionality
+Zarf is designed for airgapped environments, meaning that all functionality should operate without requiring external dependencies. This includes:
+- Ensuring that all packaged software and dependencies are self-contained.
+- Avoiding reliance on external repositories or registries during runtime.
+- Providing a seamless user experience for both online and offline operations.
+
+## 2. Minimal and Secure Binary
+The Zarf binary must remain lightweight and secure while offering core functionality. Considerations include:
+- Baking in only essential tools necessary for package management, deployment, and validation.
+- Avoiding unnecessary bloat by carefully selecting embedded utilities.
+- Ensuring security best practices, including dependency verification and signed releases.
+
+## 3. Clear Scope Within Kubernetes Environments
+Zarf operates in Kubernetes clusters but does not attempt to replace existing Kubernetes tooling. Instead, Zarf should:
+- Focus on enabling package management and deployment in airgapped clusters.
+- Complement, rather than compete with, Kubernetes-native solutions (e.g., Helm, Kustomize, and Operator patterns).
+- Facilitate cluster initialization and ongoing software management without enforcing opinionated infrastructure choices.
+
+## 4. Declarative and Reproducible Workflows
+To maintain consistency in deployments, Zarf should emphasize:
+- Declarative package definitions that allow for repeatable deployments.
+- Versioned and auditable package formats that ensure integrity across different environments.
+- Support for GitOps workflows where applicable.
+
+## 5. Operational Simplicity
+Zarf should remain simple to use and operationalize, with considerations such as:
+- Clear and concise configuration files that reduce complexity.
+- A user-friendly CLI that abstracts away unnecessary details while allowing expert-level control when needed.
+- Automation and scripting capabilities that streamline usage in constrained environments.
```

Comment:
Developer simplicity I think is also key here - there are special things like the `zarf dev find-images` commands that really help developers build up their packages and gather resources.